org.pygame.Pygame.BaseApp.json

```diff
@@ -0,0 +1,116 @@
+{
+    "id": "org.pygame.Pygame.BaseApp",
+    "command": "/usr/bin/bash",
```

Comment:
What do you need it for?

Reply:
I don't think I did need it - it was probably leftover from earlier experimentation.